### DISCUSSION TITLE
Security Issue - CVE2020-11022

### DISCUSSION THREAD

**Discussion by Jason-Hsu** [quay/quay#1401] [Q&A] [answered]
Hi,

We're using [Quay](https://github.com/quay/quay). We use Nessus Professional Version 8 to scan, and found an issue. It indicates [NVD - cve-2020-11022 (nist.gov)](https://nvd.nist.gov/vuln/detail/cve-2020-11022). Is there a plan to fix this issue by upgrading jquery? Thanks for your help.

**Comment by dmesser** [comment, marked as answer]
Thanks for raising this @Jason-Hsu. We are not looking to fix this issue because the security risk is minimal but upgrading jQuery will cause tons of work required in the current Quay UI. Meanwhile we are working on a new UI that is not subject to vulnerable version of jQuery over at [quay/quay-ui](https://github.com/quay/quay-ui).

We performed an analysis of various jQuery related XSS CVEs and determined that they posed minimal risk of compromise to a Quay installation.  The Javascript libraries in question are only used within the Quay web interface and while an attacker could exploit these vulnerabilities it would do nothing to compromise the data or runtime integrity of Quay itself.  This is because the Quay web interface relies exclusively on Quay’s own secure API layer for all of its functionality.  Quay’s API layer is automatically secured with authentication and CSRF checks by default.
